packages/svelte/README.md

```diff
@@ -0,0 +1,42 @@
+# `@carbon/charts-svelte`
+
+> Carbon Charting Svelte Wrappers
+
+**[Storybook demos](https://carbon-design-system.github.io/carbon-charts/svelte)**
+
+**[Storybook demo sources](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core/demo/data)**
+
+## Getting started
+
+Run the following command using [npm](https://www.npmjs.com/):
+
+```bash
+npm install -S @carbon/charts @carbon/charts-svelte d3
+```
+
+If you prefer [Yarn](https://yarnpkg.com/en/), use the following command
+instead:
+
+```bash
+yarn add @carbon/charts @carbon/charts-svelte d3
+```
+
+## Step-by-step instructions
+
+TBA
+
+## Codesandbox examples
+
+TBA
```

Comment:
I've updated the READMEs for packages. We should do the same here

Reply:
I've filled in the Codesandbox examples sections in the README.

However, the "Step-by-step" section is still `TBA`. Should I add in a few steps for basic usage?

Reply:
hmmm... yeah maybe at least in the README just add that